e2e/steps/home.steps.ts

```diff
@@ -0,0 +1,20 @@
+import {World, expect, Callback} from '../support/config';
```

Comment:
Definitely! And more, `World` could be a class instead of a function, right? Gonna try this out..